types/ioredis/index.d.ts

```diff
@@ -335,6 +336,12 @@ declare module IORedis {
         hscan(args: any[], callback?: ResCallbackT<any>): any;
         zscan(...args: any[]): any;
         zscan(args: any[], callback?: ResCallbackT<any>): any;
+        pfmerge(...args: any[]): any;
+        pfmerge(args: any[], callback?: ResCallbackT<any>): any;
```

Comment:
The second overload isn't going to be reachable since `...args: any[]` accepts anything.

Reply:
It's done, thanks for the review 👍